modules/external-api/src/main/java/org/opencastproject/external/endpoint/GroupsEndpoint.java

```diff
@@ -24,9 +24,12 @@
 import static com.entwinemedia.fn.data.json.Jsons.f;
 import static com.entwinemedia.fn.data.json.Jsons.obj;
 import static com.entwinemedia.fn.data.json.Jsons.v;
+import static javax.servlet.http.HttpServletResponse.SC_FORBIDDEN;
+import static javax.servlet.http.HttpServletResponse.SC_NOT_FOUND;
 import static org.apache.commons.lang3.StringUtils.join;
 import static org.apache.commons.lang3.exception.ExceptionUtils.getMessage;
 import static org.apache.commons.lang3.exception.ExceptionUtils.getStackTrace;
+import static org.apache.http.HttpStatus.SC_CONFLICT;
```

Comment:
Any special motivation to prefer this over javax.servlet.http.HttpServletResponse.SC_CONFLICT?

Seems a bit weird to grab the constant from different classes...

Reply:
Thanks for seeing that, I'll fix it to be consistent.